src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/features/HandleZipTestDummy.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.features;
+
+import java.util.List;
+import java.util.function.BiConsumer;
+import java.util.stream.Collectors;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipFile;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class HandleZipTestDummy implements BiConsumer<String, ZipFile> {
+
+    private boolean invoked = false;
+    private String jurisdiction;
+    private List<String> pdfNames;
+
+    @Override
+    public void accept(String s, ZipFile zipFile) {
+        invoked = true;
+        jurisdiction = s;
+        pdfNames = zipFile.stream().map(ZipEntry::getName).collect(Collectors.toList());
+    }
+
+    void zipContainsFiles(String... fileNames) {
+        assertThat(pdfNames).containsExactlyInAnyOrder(fileNames);
+    }
+
+    void jurisdictionNameIs(String name) {
+        assertThat(jurisdiction).isEqualTo(name);
+    }
+
+    void isInvoked() {
+        assertThat(invoked).isTrue();
+    }
```

Comment:
As mentioned before, let's keep assertions in test. All those method are only called in one place.